src/basic/src/ScalarSequence.C

```diff
@@ -1729,14 +1729,14 @@ ScalarSequence<T>::subBasicHistogram(
   for (unsigned int j = 0; j < dataSize; ++j) {
     double value = m_seq[j];
     if (value < minHorizontalValue) {
-      bins[0] += value;
+      bins[0]++;
```

Comment:
To be clear, the first commit this change shows up in is d23705cf2871e0f66810ecbfbef4092236a1e366